src/Axis.js

```diff
@@ -427,6 +427,7 @@ export default class Axis extends BaseClass {
         }
       }
     }
+    else if (horizontal) this._rotateLabels = this._labelRotation;
```

Comment:
Is this `this._rotateLabels` variable used in any other files/functions? If not, why not just make it a local variable (not attached to `this`).